packages/strapi-plugin-content-manager/admin/src/containers/Initializer/index.js

```diff
@@ -21,11 +21,18 @@ const Initializer = ({ updatePlugin }) => {
 
       try {
         const { data } = await request(requestURL, { method: 'GET' });
-
         const menu = [
           {
             name: 'Content Types',
-            links: data,
+            links: data.filter(
```

Comment:
Maybe you can use [`lodash.groupBy`](https://lodash.com/docs/4.17.15#groupBy) so you avoid filtering the result twice. I haven't tested it but I think it would simplify it a bit.